app/tests/util/member_personas.py

```diff
@@ -80,6 +83,69 @@ async def bob_and_alice_public_did(
     alice_acapy_client: AcaPyClient,
     bob_acapy_client: AcaPyClient,
 ) -> BobAlicePublicDid:
+    tenant_admin_acapy = governance_acapy_client()
```

Comment:
tenant admin is the admin user of the tenant agent, while this is the governance agent. To avoid confusion maybe we can call it governance_admin_acapy?

```suggestion
    governance_admin_acapy = governance_acapy_client()
```